lib/index.js

```diff
@@ -58,14 +62,23 @@ function createHealthFood(){
 }
 
 function healthFoodDraw(healthFood){
-  context.beginPath();
-  context.fillStyle = healthFood.color;
-  context.fillRect(healthFood.x, healthFood.y, healthFood.width, healthFood.height);
+  if (healthFood.type === 'beet'){
```

Comment:
Let's ask about why the other version of this didn't work at our check in tomorrow